tools/labs/templates/hello_assignment/sol/hello-world.c

```diff
@@ -0,0 +1,21 @@
+#include <linux/kernel.h>
```

Comment:
Please run ./scripts/checkpatch.pl -f --strict hello.c to verify for coding style problems